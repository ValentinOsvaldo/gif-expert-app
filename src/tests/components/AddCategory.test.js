import { shallow } from 'enzyme';
import AddCategory from '../../components/AddCategory';

describe( 'Pruebas en <AddCategory/>', () => {

    const setCategories = () => {}
    const wrapper = shallow(<AddCategory setCategories={ setCategories } />);

    test( 'debe de mostrarse correctamente', () => {
        expect( wrapper ).toMatchSnapshot();
    } );

    test( 'Debe de cambiar la caja de texto', () => {
        const input = wrapper.find('input');

        input.simulate('change');
    } );

} );