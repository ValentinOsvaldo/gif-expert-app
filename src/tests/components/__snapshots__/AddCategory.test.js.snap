// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <AddCategory/> debe de mostrarse correctamente 1`] = `ShallowWrapper {}`;
